buildlib/azure-pipelines-pr.yml

```diff
@@ -61,6 +61,67 @@ resources:
       image: rdmz-harbor.rdmz.labs.mlnx/swx-infra/x86_64/sles12sp5/builder:mofed-5.0-1.0.0.0
       options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools
 
+    - container: centos7_cuda_11_0
```

Comment:
can we put this huge list in a separate YML file?

Reply:
it's impossible 
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/resources?view=azure-devops&tabs=schema

resources doesn't support templates
and extend: template is not applicable 

we can leave it here and for example move all stages to separate files ( like refactoring in other PR)

Reply:
ok